project_task_parent_due_auto/models/project_task.py

```
@@ -14,15 +14,17 @@ def create(self, vals_list):
-        if "date_deadline" in vals:
+        if "date_deadline" in vals or "stage_id" in vals:
             self.update_parent_task_dates()
         return res
 
     def update_parent_task_dates(self):
         for task in self:
             parent = task.parent_id
             if parent:
-                children = parent.child_ids.filtered("date_deadline")
+                children = parent.child_ids.filtered(
+                    lambda child: not child.stage_id.fold and child.date_deadline
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)